prowler/providers/aws/services/ec2/ec2_launch_template_no_secrets/ec2_launch_template_no_secrets.py

```diff
@@ -0,0 +1,61 @@
+import os
+import tempfile
+import zlib
+from base64 import b64decode
+
+from detect_secrets import SecretsCollection
+from detect_secrets.settings import default_settings
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.ec2.ec2_client import ec2_client
+
+
+class ec2_launch_template_no_secrets(Check):
+    def execute(self):
+        findings = []
+        for template in ec2_client.launch_templates:
+            report = Check_Report_AWS(self.metadata())
+            report.region = template.region
+            report.resource_id = template.id
+            report.resource_arn = template.arn
+
+            versions_with_secrets = []
+
+            for version in template.versions:
+                print(version.template_data)
```

Comment:
```suggestion
```

Reply:
Thanks, missed this while cleaning up the code.